docs/design_new.md

```diff
@@ -0,0 +1,101 @@
+# Key Transparency New Design
+
+## Objective
+
+Key Transparency currently suffers from a design-space trade-off in two dimensions between
+low latency key updates and verification bandwidth; each snapshot of the
+key-value Merkle tree creates an additional point in time that clients must
+audit when inspecting their account history.
+Using Key Transparency v0.4.0 and below requires the presence of powerful
+auditors who can verify that no information has been lost or tampered with between all snapshots.
+
+| Update Latency | History Audit `O(T * log N)`  | Key Lookup |
+|----------------|------------------|-------------|
+| hr             | 45 Kb/day         | ~2Kb       |
+| min            | 2.7 Mb/day        | ~2Kb       |
+| s              | 160 Mb/day        | ~2Kb       |
+
+Table: *Based on 1B users, updating 200 keys per second*
+
+Key Transparency v0.5 is a new set of algorithms that removes this fundamental trade-off by sharing
+the task of auditing between pairs of clients. As a result, all configurations have efficiently
+auditable data structures that do not require the presence of 3rd party auditors.
+
+We expect that many applications will want to prioritize lower latency and
+reliability by using a *trust, then verify* mode whereby clients receive key
+updates that they can later verify as being part of the global, consistent
+state.  However, this does not preclude individual clients that have a lower
+tolerance for risk from waiting for full, globally consistent verification
+before using fresh public keys from their peers.
+
+| Update Latency | History Audit `O(log T * log N)` | Key Lookup |
+|----------------|------------------|------------|
+| hr             | 18Kb             | ~2Kb       |
+| min            | 30Kb             | ~2Kb       |
+| s              | 40Kb             | ~2Kb       |
+
+
+## Data Structures
+
+### Gossip Network
```

Comment:
Maybe put this further down, before or with the "Client Verification" section.

Reply:
It's not a pure Data Structure problem, but rather belongs to a bigger communication / verification topic.

Reply:
hmm, at the moment there is a logical flow to it, but it does feel odd to begin with a TODO